webpack.config.js

```diff
@@ -189,7 +204,7 @@ module.exports = [{
     // In development, these are emitted as js files to facilitate hot module replacement. In
     // all other cases, ExtractTextPlugin is used to generate the final css, so this is given a
     // dummy ".css-entry" extension.
-    filename: '[name].' + (IS_PROD ? 'min.' : '') + 'css' + (IS_DEV ? '.js' : '-entry'),
+    filename: CSS_JS_FILENAME_OUTPUT_PATTERN,
```

Comment:
The comment preceding this line mentions a dummy `.css-entry` extension but that doesn't seem to actually be employed anymore. Does it still need to be? Otherwise the comment needs to be updated.